packages/contracts/resource/dev.toml

```diff
@@ -7,7 +7,7 @@
 # blockchain id, e.g. baseSepolia=84532, arbSepolia=421614
 chain_id = 84532
 
-[evm.contracts]
+[evm.address]
 # credbull-devops wallets.  wallet numbers are 1-based (as opposed to 0-based in anvil)
 # devops deployer (wallet 5)
 deployer.private_key = ""
```

Comment:
@lucasia Is this key used? Or accidentally left behind?

Reply:
it looks safe to be deleted.

i think we can even delete the "dev" environment once we merge in the SDK.  no need to manage so many github secrets anymore.

Reply:
Deleted the key.